storage-service/src/test/java/com/stnetix/ariaddna/persistence/TestHelper.java

```diff
@@ -62,7 +62,7 @@ private static MetafileDTO getMetafileDTO(String fileUuid, String parentFileUuid
         MetafileDTO metafileDTO = new MetafileDTO();
         metafileDTO.setFileUuid(fileUuid);
         metafileDTO.setVersion(MavenUtil.getCurrentVersion());
-        metafileDTO.setAllocationStrategy(AllocationStrategy.HIGH_AVAILABILITY);
```

Comment:
Why you rename "HIGH_AVAILABILITY" to "HIGH"? 
"HIGH" it is not good abbreviation because have many meanings. 
In IT documentation always "HIGH_AVAILABILITY" -> "HA"

If this abbreviation get from specification then need change it in specification.

Reply:
I rename it to HA